lib/nodes/fpga.cpp

```diff
@@ -341,6 +341,11 @@ int FpgaNode::slowWrite(Sample *smps[], unsigned cnt) {
 }
 
 std::vector<int> FpgaNode::getPollFDs() {
+  if (!lowLatencyMode && card && !card->polling) {
+    return card->vfioDevice->getEventfdList();
+  } else {
+    return {};
+  }
   return card->vfioDevice->getEventfdList();
```

Comment:
Seems like this is dead code? Can we remove it? Or just move the else branch out?